Userland/Libraries/LibGfx/ImageFormats/BilevelImage.cpp

```diff
@@ -292,11 +292,41 @@ ErrorOr<NonnullRefPtr<BilevelImage>> BilevelImage::create_from_bitmap(Gfx::Bitma
             }
         }
         break;
+    case DitheringAlgorithm::Clustered4x4:
```

Comment:
Do we want even-sized matrices for this?
I would expect better results with a matrix with a centered zero.